src/language-css/parser-postcss.js

```diff
@@ -554,7 +554,8 @@ function parseNestedCSS(node, options) {
         // Remove unnecessary spaces in SCSS variable arguments
         params = params.replace(/(\$\S+?)\s+?\.{3}/, "$1...");
         // Remove unnecessary spaces before SCSS control, mixin and function directives
-        params = params.replace(/^(?!if)(\S+)\s+\(/, "$1(");
+        // Move spaces after the `(`, so we can keep the range correct
+        params = params.replace(/^(?!if)(\S+)(\s+)\(/, "$1($2");
```

Comment:
We have very dirty solutions... Bu I agree, it is impossible to fix right now, so let's merge

Reply:
The line above this may also cause problems, but let's wait for someone to report 😄 

Hope this trick won't break other cases.